projects/lionheart/Plan.cpp

```diff
@@ -41,6 +41,13 @@ lionheart::Paths::Paths(std::shared_ptr<const Map> const& map, int maxSpeed) : v
 {
   std::vector<lionheart::Direction> const dirs{
     lionheart::Direction::NORTH, lionheart::Direction::EAST, lionheart::Direction::SOUTH, lionheart::Direction::WEST};
+  
+  //loading bar part 1
+  printf("\033[1A"); //steps up on line (unique to linux but there are windows solutions)
```

Comment:
This project must remain cross platform so without the appropriate guards to keep this only on ansi displays I can't accept it.